auxiliary/dime-analytics-resources.tex

```diff
@@ -0,0 +1,142 @@
+suitable for users of varying levels of expertise,
+up-to-date with the latest technological advances
+in electronic data collection,
+and curated by a vibrant network of editors
+with expertise in the field.
+
+\resourcepar\textbf{Stata Visual Library.\sidenote{
+		\url{https://worldbank.github.io/Stata-IE-Visual-Library/}}}
+A curated, easy-to-browse selection of graphs created in Stata.
+Clicking on each graph reveals the source code,
+to allow for easy replication.
+
+\resourcepar\textbf{R Econ Visual Library.\sidenote{
+		\url{https://worldbank.github.io/r-econ-visual-library/}}}
+A curated, easy-to-browse selection of graphs created in R.
+Clicking on each graph reveals the source code,
+to allow for easy replication.
+
+\resourcepar\textbf{DIME Analytics Research Standards.\sidenote{
+		\url{https://github.com/worldbank/dime-standards}}}
+A repository outlining DIME's public commitments to
+research ethics, transparency, reproducibility,
+data security and data publication,
+along with supporting tools and resources.
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+\section{Flagship Courses}
+
+\textbf{Manage Successful Impact Evaluations (MSIE).\sidenote{
+		\url{https://osf.io/h4d8y/}}}
+DIME Analytics’ flagship training is a week-long annual course,
+held in-person in Washington DC.
+MSIE is intended to improve the skills and knowledge of
+impact evaluation (IE) practitioners,
+familiarizing them with critical issues in
+IE implementation, recurring challenges,
+and cutting-edge technologies.
+The course consists of lectures and hands-on sessions.
+Through small group discussions and interactive computer lab sessions,
+participants work together to apply what they’ve learned
+and have a first-hand opportunity to develop skills.
+Hands-on sessions are offered in parallel tracks,
+with different options based on software preferences and skill level.
+
+\resourcepar\textbf{Manage Successful Impact Evaluation Surveys (MSIES).\sidenote{
+		\url{https://osf.io/resya/}}}
+A fully virtual course,
+in which participants learn the workflow for primary data collection.
+The course covers best practices at all stages of the survey workflow,
+from planning to piloting instruments
+and monitoring data quality once fieldwork begins.
+There is a strong focus throughout on research ethics and reproducible workflows.
+The course uses a combination of virtual lectures,
+case studies, readings, and hands-on exercises.
+
+\resourcepar\textbf{Research Assistant Onboarding Course.\sidenote{
+		\url{https://osf.io/fqhdt/}}}
+This course is designed to familiarize Research Assistants and Research Analysts
+with DIME’s standards for data work.
+By the end of the courses six sessions,
+participants will have the tools and knowledge to
+implement best practices for transparent and reproducible research.
+The course will focus on how to set up a collaborative workflow for
+code, datasets and research outputs.
+Most content is platform and software agnostic,
```

Comment:
wont this look a bit weird?

Reply:
@bbdaniels said this in chat:

>I think that is a standard construction? It can be better though by writing more, like “platform-independent and software-agnostic”

@mariaruth , do you want to be the tie-braker?

Reply:
I'm ok either way. Construction as Ben had it is correct but I like platform-independent and software-agnostic.

Reply:
ok, I will commit "platform-independent and software-agnostic"